io.github.francivaldo4334.Bitsafe_cli.yaml

```
@@ -0,0 +1,30 @@
+id: io.github.francivaldo4334.Bitsafe_cli
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: bitsafe.sh
+modules:
+  - python3-requirements.json
+  - name: app-install
```

the name is not explicit.
```suggestion
  - name: bitsafe
```